### PR TITLE
Refactor User Avatar

### PR DESCRIPTION
**Pull request by ameerul-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.
Screenshot 2023-05-30 at 11 20 52 AM

### CODE REVIEW

File: packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 import { DesktopWrapper, MobileWrapper, Text } from '@deriv/components';
 import { observer } from 'mobx-react-lite';
-import UserAvatar from 'Components/user/user-avatar/user-avatar.jsx';
+import UserAvatar from 'Components/user/user-avatar';
 import { useStores } from 'Stores';
 import { daysSince, isMobile } from '@deriv/shared';
 import { Localize } from 'Components/i18next';
 import TradeBadge from 'Components/trade-badge';
-import MyProfilePrivacy from '../my-profile-privacy';
+import MyProfilePrivacy from 'Pages/my-profile/my-profile-stats/my-profile-privacy/my-profile-privacy.jsx';
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx]:
```typescript
import MyProfilePrivacy from 'Pages/my-profile/my-profile-stats/my-profile-privacy/my-profile-privacy
```

File: packages/p2p/tsconfig.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
             "Assets/*": ["src/assets/*"],
             "Components/*": ["src/components/*"],
             "Constants/*": ["src/constants/*"],
+            "Pages/*": ["src/pages/*"],
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/tsconfig.json:9]: Hmm nice!!, i forgot to put this here 🙏

File: packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 import StarRating from 'Components/star-rating';
 import RecommendedBy from 'Components/recommended-by';
-import BlockUserCount from 'Pages/advertiser-page/block-user/block-user-count';
+import BlockUserCount from 'Pages/advertiser-page/block-user/block-user-count.jsx';
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx]:
```typescript
import BlockUserCount from 'Pages/advertiser-page/block-user/block-user-count
```

**Review by adrienne-deriv — APPROVED** [review]
LGTM

**Review by farhan-nurzi-deriv — CHANGES_REQUESTED** [review]

**Review by farhan-nurzi-deriv — APPROVED** [review]

**Pull request merged**